packages/@okta/vuepress-theme-prose/layouts/AuthorizationLayout.vue

```diff
@@ -19,9 +19,14 @@
 </template>
 
 <script>
+import pendo from '../util/pendo'
```

Comment:
To resolve your SSR issues, you need to import this within the mounted function if pendo is accessing the window object on import just like you did for `codemirror`